components/dashboard/upload/components/UploadTabBar.tsx

```diff
@@ -1,18 +1,31 @@
 'use client';
 
-import React, { Dispatch, SetStateAction } from 'react';
+import React, { Dispatch, SetStateAction, useState } from 'react';
 
 import { dashBoardTabType } from '@/types/dashboard';
 
 import UploadTabBarContainer from './ui/UploadTabBarContainer';
 
+type ArrayLiteral<T extends ReadonlyArray<any>> = T[number];
 export interface UploadTabBarProps {
-  category: dashBoardTabType;
-  setCategory: Dispatch<SetStateAction<dashBoardTabType>>;
+  categoryListData: CategoryListProps[];
+  currentCategory: string[];
+  setCategory: Dispatch<SetStateAction<string>>;
 }
 
 const UploadTabBar = (props: UploadTabBarProps) => {
-  return <UploadTabBarContainer categoryProps={props} />;
+  const category = [...props.currentCategory] as const;
```

Comment:
```suggestion
  const {currentCategory : [category]} = props;
```

이거랑 같은 코드인가?!

Reply:
그렇다면
```suggestion
   const {currentCategory : [category], categoryListData, setCategory} = props;
```
싹다 구조분해할당 가능할듯?!